scenarios/scenario-10/wso2-apim.yaml

```diff
@@ -0,0 +1,23 @@
+#Copyright (c)  WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+apiVersion: apim.wso2.com/v1alpha1
+kind: APIManager
+metadata:
+  name: apim-pattern-2
+spec:
+  pattern: Pattern-2
+  enableAnalytics: 'true'
```

Comment:
By default we can set enableAnalytics to true and not need to define this property here. It should be an optional property.